.github/workflows/ci.yml

```
@@ -122,7 +122,21 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v5
         with:
-          dotnet-version: '8.0.x'
+          dotnet-version: '9.0.x'
+
+      - name: Build Mystira.Core
+        id: build-core
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:3:24: Double quote to prevent globbing and word splitting

(shellcheck)